frontend/public/components/utils/dropdown.jsx

```diff
@@ -364,7 +367,10 @@ Dropdown.propTypes = {
   dropDownClassName: PropTypes.string,
   enableBookmarks: PropTypes.bool,
   headerBefore: PropTypes.objectOf(PropTypes.string),
-  items: PropTypes.object.isRequired,
+  items: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.array,
```

Comment:
Items is still always a map, right?

Reply:
Correct, eliminating this.  A change I made earlier when I was going down the road of allowing an array.